src/index.html

```diff
@@ -15,8 +20,50 @@
       rel="stylesheet"
       href="./style.css"
     />
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!--Header start-->
+    <header class="header__menu">
+      <div class="wrapper">
+        <div class="header__wrapper">
+                  <a href="#!" class="header__logo-link">
+                  <img src="./images/logo.png" alt="MOYO logo" class="header__logo-pic">
+                  </a>
+
+          <nav class="header__nav">
+            <ul class="header__list">
+                <li class="header__item">
```

Comment:
Fix classes everywhere
```suggestion

          <nav class="nav">
            <ul class="nav__list">
                <li class="nav__item">
```